frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/use-embedding-settings-icon-colors.tsx

```diff
@@ -0,0 +1,19 @@
+export type UseEmbeddingSettingsIconColorsProps = {
```

Comment:
(context: sometimes I need to have so many files open to see what's going on that I'm starting to wonder if many small files really help)
nit:I wonder if the two icons could live in the same file, so that this didn't need to be exported

Reply:
there's three icons 😅 I think once we go past 2 components then we should probably start splitting


On another note, could we use those icons in the embedding modal?

Reply:
Are those the exact same icons?
The issue is that this is going to a feat branch and mine are supposed to go directly to master 😰

Reply:
oh no

Reply:
yep, they're the same icons. Let's figure this out when we have a little time to step back

Reply:
(my prs are not going to master in the end, so we can fix it later in the branch)

Reply:
Some initial thoughts.
- I wouldn't make this a hook since we're not using any of React's features here and making it a hook makes me guess that it might use other Hooks e.g. getting some instance color from a selector which is not the case.
- I'd love to see these colors stay in CSS module, if I read the previous convos correctly, we remove it because of the CSS specificity hack, but that could easily be eliminated by porting `Label` component to using CSS modules.